GenderPayGap.WebUI/Views/Organisation/ScopeDeclared.cshtml

```diff
@@ -1,18 +1,18 @@
 ﻿@{
     ViewBag.Title = "We need more information - Gender pay gap reporting service";
+    Layout = "~/Views/GovUkFrontend/GovUkFrontendLayout.cshtml";

```

Comment:
Does this cause any issues? (i.e. does the page look the same?)

Reply:
It's the layout page we use for most of the rest of the site, it did break everything, but I fixed them to make it look like the rest of the confirm pages

Reply:
I see, thanks 😄